modules/tensor_mechanics/src/kernels/StressDivergenceRSphericalTensors.C

```diff
@@ -70,6 +64,8 @@ StressDivergenceRSphericalTensors::calculateJacobian(unsigned int ivar, unsigned
 {
   RealGradient test_r, phi_r;
 
+  mooseAssert(ivar == 0 && jvar == 0, "Invalid component in Jacobian Calculation"); // Only nonzero case for a 1D simulation
```

Comment:
Wait - what the hell is `0` doing here?  That's going to be wrong unless `disp_r` is the first variable in the system!!!

The variable number for `disp_r` needs to be saved off from `coupled()` and compared to!